### DISCUSSION TITLE
Issue to create inventree on Synology Container

### DISCUSSION THREAD

**Discussion by HKGEric** [inventree/InvenTree#5053] [Q&A] [answered]
I tried to create inventree docker in synology DS923+, i reach the following error when starting up inventree service. 
Error:
<img width="1103" alt="image" src="https://github.com/inventree/InvenTree/assets/89208844/275f8df4-9130-421e-8612-a1834f57bccc">

Configuration:
<img width="500" alt="image" src="https://github.com/inventree/InvenTree/assets/89208844/7ac96c16-fecd-4b6e-b894-22e2a55b5057">
![image](https://github.com/inventree/InvenTree/assets/89208844/4bcb3467-98b9-4fcf-8695-b024e18e9a12)

**Comment by wolflu05** [comment]
Have you run the invoke update task?

**Reply by HKGEric** [reply]
If i ran invoke update in DS923 via ssh , it will lost the database configuration and create new container.
<img width="638" alt="image" src="https://github.com/inventree/InvenTree/assets/89208844/068e56b2-03b9-4c65-b519-dd596b858b7a">

**Reply by wolflu05** [reply]
You need to either run this inside the existing container or you would need to also use all the envs and volumes like your normal container.

**Reply by HKGEric** [reply]
Just create number of container to execute invoke update take. Database created successfully. Service can be start but service to look up folder static.
<img width="600" alt="image" src="https://github.com/inventree/InvenTree/assets/89208844/2258dcfa-cac3-4945-897c-9976377a60d2">

**Reply by wolflu05** [reply, marked as answer]
Where have you run the invoke update, it also needs the volumes, not only the db.

**Reply by HKGEric** [reply]
Thanks. I found the missing volumes setting in invoke update.. Not inventree is working normally.
<img width="738" alt="image" src="https://github.com/inventree/InvenTree/assets/89208844/32fe5803-9785-464a-ad55-607e388f1b45">

**Comment by matmair** [comment]
@HKGEric deploying to synology directly turned out pretty difficult. I use Portainer for it nowadays. In theory it should be much easier with DSM 7.2 as that supports docker compose.

What you could try right now: set INVENTREE_AUTO_UPDATE to true in your variables. That will run migrations automatically.

**Reply by HKGEric** [reply]
tried to configure INVENTREE_AUTO_UPDATE to true still with no luck
